Resources/input/fsh/profiles/6.3_EPrescriptionTokenIdentifier.fsh

```diff
@@ -0,0 +1,33 @@
+Profile: GEM_PR_MEDREQ_EPrescriptionTokenIdentifier
+Parent: Identifier
+Id: GEM-PR-MEDREQ-EPrescriptionTokenIdentifier
+Title: "GEM PR MEDREQ E Prescription Token Identifier"
+Description: "Identifier für den E-Rezept Token"
+* insert Meta (GEM-PR-MEDREQ-EPrescriptionTokenIdentifier)
+* system 1..1 MS
+* system = "https://gematik.de/fhir/erp/sid/GEM_NS_EPrescriptionToken" (exactly)
+* value 1..1 MS
+//TODO: fix Expression * value obeys GEM-INV-E-Prescription-Regex
+
+Invariant: GEM-INV-E-Prescription-Regex
+Description: "Regex Invariante zur Validierung eines E-Rezept Tokens"
+Expression: "$this.toString().matches('^Task/[0-9]{3}.[0-9]{3}.[0-9]{3}.[0-9]{3}.[0-9]{3}.[0-9]{2}/$accept?ac=[A-Za-z0-9]+$')"
```

Comment:
@simoneOnFhir 
Kannst du bitte diesen regex für FHIR funktionierend machen? 
Hier ein Beispiel Token: Task/160.000.000.000.123.76/$accept?ac=777bea0e13cc9c42ceec14aec3ddee2263325dc2c6c699db115f58fe423607ea

Reply:
hier haben ein paar escapes gefehlt, ich pushe gleich einen Fix

Reply:
Ich habe keine Push Rechte, darum hier die korrigierte Version:
`Expression: "$this.toString().matches('^Task\/[0-9]{3}\.[0-9]{3}\.[0-9]{3}\.[0-9]{3}\.[0-9]{3}\.[0-9]{2}\/\$accept\?ac=[A-Za-z0-9]+$')"`

Reply:
Danke @patrick-werner auch hier. 
Wenn ich deine Expression so in eine .fsh file poste, dann meldet SUSHI folgenden Fehler:
"mismatched input '"$this.toString().matches('^Task\/[0-9]{3}\.[0-9]{3}\.[0-9]{3}\.[0-9]{3}\.[0-9]{3}\.[0-9]{2}\/\$accept\?ac=[A-Za-z0-9]+$"' expecting STRING"

Reply:
Ich teste das Problem heute nochmals durch, FSH macht manchmal escape Problem. (Es wird aber an einer einfacheren Lösung gearbeitet: https://chat.fhir.org/#narrow/stream/215610-shorthand/topic/Proposed.20new.20syntax.20option.20for.20insert.20rules